<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChercherContact</class>
 <widget class="QWidget" name="ChercherContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contacts</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Rechercher Nom :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Rechercher Entreprise :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="LineNom"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="LineEntreprise"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="PBLancer">
       <property name="text">
        <string>Lancer la Recherche</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="PBListeComplete">
       <property name="text">
        <string>Liste Complète</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Rechercher Date :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateTimeEdit" name="RDate"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Nombre de Contact :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="NbContact">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Trier par ordre :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PBAlphabet">
       <property name="text">
        <string>Alphabétique</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PBDate">
       <property name="text">
        <string>Date de Création</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="Table2"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
